src/api/models/api_response.py

```diff
@@ -0,0 +1,108 @@
+# coding: utf-8
+"""Assessment Model."""
+
+from __future__ import absolute_import
+
+# cisagov Libraries
+from api import util
+from api.models.base_model_ import Model
+
+# from datetime import date, datetime  # noqa: F401
+# from typing import Dict, List  # noqa: F401
+
+
+class ApiResponse(Model):
+    """NOTE: This class is auto generated by the swagger code generator program.
+
+    Do not edit the class manually.
+    """
+
+    def __init__(
+        self, code: int = None, type: str = None, message: str = None
+    ):  # noqa: E501
+        """Initialize the ApiResponse model defined in Swagger.
+
+        :param code: The code of this ApiResponse.   noqa: E501
+        :type code: int
+        :param type: The type of this ApiResponse.   noqa: E501
+        :type type: str
+        :param message: The message of this ApiResponse.  # noqa: E501
+        :type message: str
+        """
+        self.swagger_types = {"code": int, "type": str, "message": str}  # type: ignore
+
+        self.attribute_map = {
+            "code": "code",
+            "type": "type",
+            "message": "message",
+        }  # type: ignore
```

Comment:
The dict types can be easily defined and should not require `# type: ignore`.  For more info, see [`typing.TypedDict`](https://docs.python.org/3/library/typing.html#typing.TypedDict).  For a bunch of other typing examples, take a look [here](https://github.com/cisagov/publish-egress-ip-lambda/blob/first-commits/src/lambda_handler.py) at something I recently did (still under review, but should be good for reference).